chia/full_node/full_node.py

```diff
@@ -1387,7 +1458,7 @@ async def prevalidate_blocks(
             blocks_to_validate,
             self.blockchain.pool,
             {},
-            copy.copy(vs),
+            vs,
```

Comment:
I would update the docstring w.r.t. `ValidationState` object.

Reply:
Now that I'm looking at this, I'm tempted to remove this (very thin) wrapper function. It seems like it does more harm than good.
But I'll hold off on that for a future PR. For now I'll add comments